R/README.md

```diff
@@ -11,4 +11,4 @@ Install from Github with:
 devtools::install_github("wesm/feather/R")
 ```
 
-Feather uses C++11, so if you're on windows, you'll require the [experimental gcc 4.93 toolchain](https://github.com/rwinlib/r-base/wiki/Testing-Packages-with-Experimental-R-Devel-Build-for-Windows). This should become official when R 3.3.0 is released, which is also when we'll submit it to CRAN, so installation will eventually be painless.
+Feather uses C++11, so if you're on Windows, you'll need the [experimental gcc 4.93 toolchain](https://github.com/rwinlib/r-base/wiki/Testing-Packages-with-Experimental-R-Devel-Build-for-Windows). This toolchain should be part of R 3.3.0 when it's released, which is also when we'll submit it to CRAN, so installation will eventually be painless.
```

Comment:
OK. How about the "it" in the phrase "which is also when we'll submit it to CRAN"? Does that refer to the toolchain or to Feather?